egs/zeroth_korean/s5/local/chain/tuning/run_tdnn_1a.sh

```diff
@@ -0,0 +1,277 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+# First the options that are passed through to run_ivector_common.sh
+# (some of which are also used in this script directly).
+stage=0
+nj=30
+train_set=train_clean
+speed_perturb=true
+test_sets="test_clean"
+gmm=tri4        # this is the source gmm-dir that we'll use for alignments; it
+                 # should have alignments for the specified training data.
+nnet3_affix=       # affix for exp dirs, e.g. it was _cleaned in tedlium.
+
+# Options which are not passed through to run_ivector_common.sh
+affix=1a  #affix for TDNN directory e.g. "1a" or "1b", in case we change the configuration.
+common_egs_dir=
+
+# LSTM/chain options
+train_stage=-10
+xent_regularize=0.1
+max_param_change=2.0
+
+# training chunk-options
+get_egs_stage=-10
+chunk_width=150,110,100
+
+# training options
+num_jobs_initial=3
```

Comment:
Apart from remove_egs, I prefer that you put the rest of these (num-jobs, num-epochs, minibatch-size, learning rates), directly as args to the script; there's no need for variables.  The same goes for chunk_width and max_param_change and xent_regularize (xent_regularize should probably be zero).

Reply:
oh sorry I was mixing it up. xent_regularize should be 0.1.  It's chain-regularize that we are now making zero (in setups with l2).